stock_lot_production_date/README.rst

```diff
@@ -67,6 +67,7 @@ Contributors
 ------------
 
 -  Sébastien Alix <sebastien.alix@camptocamp.com>
+-  SodexisTeam <dev@sodexis.com>
```

Comment:
Please, remove the contributor change but add as maintainer

I mean, add a input in the `__manifest__.py` file with:
`"maintainers": ["SodexisTeam"],`

Reply:
Also the change in the index.html

Reply:
@SodexisTeam

Reply:
@moylop260 This is not correct.

The philosophy behind "maintainers" is that should use individual github accounts, not organization ones.

Reply:
@rousseldenis We are going to follow your advice and start using individual account.

Reply:
@SodexisTeam yes, indeed, declaring a PR with organization account seems OK, but contributors and maintainers should rely on individual accounts. The company can appear on authors part when applicable.

Reply:
> @rousseldenis This is not correct.

Thank you for your advice, I didn't know it

Could you refer your advice, please? (I didn't find any related with this "not correct")

> We are going to follow your advice and start using individual ACC

@SodexisTeam will you change it the value in maintainer?

Reply:
@moylop260 We are working on it.